lighthouse-core/lib/i18n/i18n.js

```diff
@@ -473,6 +473,19 @@ function replaceIcuMessageInstanceIds(inputObject, locale) {
   return icuMessagePaths;
 }
 
+/** @typedef {import('./locales').LhlMessages} LhlMessages */
+
+/**
+ * Populate the i18n string lookup dict with locale data
+ * Used when the host environment selects the locale and serves lighthouse the intended locale file
+ * @see https://docs.google.com/document/d/1jnt3BqKB-4q3AE94UWFA0Gqspx8Sd_jivlB7gQMlmfk/edit
+ * @param {LH.Locale} locale
```

Comment:
this threatens the little `LH.Locale` ~house of cards~ types. We should either limit to overriding existing locales (verify that `locale` here is actually of type `LH.Locale`) or get rid of `LH.Locale` and type all our locale names as `string`. There might be some side effects of the second option, though, if we depend on that behaving well.

Reply:
>  We should either limit to overriding existing locales

in reality this will only be used to override existing locales

Reply:
> in reality this will only be used to override existing locales

This is partly what https://github.com/GoogleChrome/lighthouse/pull/9638#discussion_r321013473 is touching on. I don't think we can treat it as a private api